src/util/eventHandlers.tsx

```diff
@@ -4,57 +4,113 @@ import { getCaretCoordinates } from "./index";
 
 export function getHandlers({
   ref,
-  suggestions,
-  setShowSuggestion,
-  showSuggestion,
+  loadSuggestions,
   setFocusIndex,
   focusIndex,
-  setCaret
+  setCaret,
+  setSuggestions,
+  mentionState,
+  setMentionState,
+  value
 }) {
   const handleSuggestionSelected = (index: number) => {
-    if (suggestions) {
-      insertText(ref?.current, suggestions[index].value);
-      setShowSuggestion(false);
+    if (loadSuggestions) {
+      insertText(ref?.current, loadSuggestions[index].value);
```

Comment:
`loadSuggestions` is a function

<img width="587" alt="image" src="https://user-images.githubusercontent.com/19513289/174980310-69b98d9d-4eb5-4781-b3f8-8da1cf4a9450.png">